### PR TITLE
Add meta information about conference

### PR DESCRIPTION
**Pull request by patrick91**
This PRs add info like introduction and generic copy text, like the screenshot :)

image

### CODE REVIEW

File: backend/conferences/models/conference.py [hunk 1]
```diff
@@ -32,6 +32,7 @@ class Conference(TimeFramedModel, TimeStampedModel):
         _("longitude"), max_digits=9, decimal_places=6, blank=True, null=True
     )
     map_link = models.URLField(_("map link"), blank=True)
+    introduction = models.TextField(_("introduction"), blank=False)
```
Comment (marcoacierno) [inline backend/conferences/models/conference.py]: Are we sure we want to make the introduction as required?

File: backend/conferences/types.py [hunk 2]
```diff
@@ -130,6 +131,14 @@ def sponsors_by_level(self, info) -> List[SponsorsByLevel]:
 
         return [SponsorsByLevel(level, list(sponsors)) for level, sponsors in by_level]
 
+    @strawberry.field
+    def copy(self, info, key: str) -> Optional[str]:
+        from cms.models import GenericCopy
```
Comment (marcoacierno) [inline backend/conferences/types.py]: Any reason for the import here?

File: backend/conferences/models/conference.py (hunk 1 above)
Comment (patrick91) [reply backend/conferences/models/conference.py]: yes, it is required on the frontend :)

File: backend/conferences/types.py (hunk 2 above)
Comment (patrick91) [reply backend/conferences/types.py]: need to double check :)

Comment (patrick91) [reply backend/conferences/types.py]: fixed :)

**Pull request merged**